resources/views/layouts/app.blade.php

```diff
@@ -10,6 +10,8 @@
 
     @include('layouts.components.assets')
 
+    <!-- font awesome -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css"/>
```

Comment:
Why is this here? Remove it and move all plugin css into **resources/views/layouts/components/assets.blade.php**

Reply:
i face problem for the reason of fontawesome.min.css not support icons. thereby i use this fontawesome from cdnjs. But your order as i remove this link.